tools/standardize_config_json.py

```diff
@@ -104,24 +103,57 @@ def print_diff(original: collections.OrderedDict, new: collections.OrderedDict):
     LOG.info("\n".join(diff))
 
 
-def process(filename: str):
-    contents = read_json_file(filename)
-    original = read_json_file(filename)
+def process(config_json_path: str, configuration: dict):
+    contents = read_json_file(config_json_path)
+    original = read_json_file(config_json_path)
 
     reordered = reorder_service(contents["properties"]["service"]["properties"])
     contents["properties"]["service"]["properties"] = reordered
 
+    sections = configuration.get("sections", {})
+    for section_name, head_and_tail in sections.items():
+        head = head_and_tail["head"]
+        tail = head_and_tail["tail"]
+        properties = contents["properties"][section_name]["properties"]
+        reordered = reorder(properties, head, tail, reorder_property)
+        contents["properties"][section_name]["properties"] = reordered
+
     print_diff(original, contents)
```

Comment:
Should this be optional?

Reply:
With 2fa054c69fc5da12528e5d47308819d78a56e74b this function will still be called unconditionally, but it will output `no changes` in case the diff was empty.

Reply:
With 2fa054c69fc5da12528e5d47308819d78a56e74b this function will still be called unconditionally, but it will output `no changes` in case the diff was empty.